R/generate_rules.R

```diff
@@ -16,6 +16,35 @@
 #'
 #' @examples
 #' TBD
-generate_rules <- function(ite_std, X, ntrees, min_nodes, max_nodes){
-  #TBD
+#'
+generate_rules <- function(X, ite_std, ntrees, min_nodes, max_nodes) {
+  # Set parameters
+  N <- dim(X)[1]
+  sf <- min(1, (11 * sqrt(N) + 1) / N)
```

Comment:
What is 11? Why it is hard coded?

Reply:
Hmm I'm not sure. I'll check with Falco on this one.